generators/cloud-init_test.go

```diff
@@ -111,19 +115,21 @@ local-hostname: {{ container.name }}
 		},
 		{
 			"vendor-data",
-			`{{ config_get("user.vendor-data", properties.default) }}
+			`{%- if config_get("cloud-init.user-data", properties.default) == properties.default -%}
+{{ config_get("user.user-data", properties.default) }}
+{%- else -%}
+{{- config_get("cloud-init.user-data", properties.default) }}
+{%- endif -%}
 `,
 			false,
 		},
 		{
 			"network-config",
-			`{% if config_get("user.network-config", "") == "" %}version: 1
-config:
-  - type: physical
-    name: {% if instance.type == "virtual-machine" %}enp5s0{% else %}eth0{% endif %}
-    subnets:
-      - type: {% if config_get("user.network_mode", "") == "link-local" %}manual{% else %}dhcp{% endif %}
-        control: auto{% else %}{{ config_get("user.network-config", "") }}{% endif %}
+			`{%- if config_get("cloud-init.network-config", "") == "" -%}
+{{ config_get("user.network-config", "") }}
+{%- else -%}
+{{- config_get("cloud-init.network-config", "") }}
+{%- endif -%}
```

Comment:
What's going on here, where is our default network config going?

Reply:
Oops. I moved that to `properties["default"]` but had forgotten to include it. Fixed now.